package.json

```diff
@@ -44,52 +44,54 @@
     "useTabs": true,
     "tabWidth": 2
   },
-  "lint-staged": {
-    "{src,test}/**/*.{css,js,json}": [
+  "simple-git-hooks": {
+    "pre-commit": "npx nano-staged"
+  },
+  "nano-staged": {
+    "**/*.{css,js,jsx,json}": [
       "prettier --write"
     ]
   },
   "devDependencies": {
-    "@babel/polyfill": "^7.7.0",
-    "@xmldom/xmldom": "^0.8.10",
```

Comment:
Moved down to `"dependencies"`, making `"devDependencies"` here dev-only stuff (not that the distinction really matters w/ build tooling but I sometimes find it helpful)